todo_app/app.py

```diff
@@ -1,26 +1,99 @@
+import os
 from flask import Flask, redirect, url_for,  render_template
 from todo_app.data import session_items
 from todo_app.flask_config import Config
 from flask import request
+import requests
 
 app = Flask(__name__)
-app.config.from_object(Config)
+app.config.from_object(Config)   
+
+KEY=os.getenv("KEY")
+TOKEN=os.getenv("TOKEN")
+    
+
+class TransportAPI:
+    def __init__(self):
+        api_keys = get_application_keys()
+        self.payload = api_keys
+
 
 @app.route('/index')
 def index():
-    list = session_items.get_items()
+    list = getAllToDoFromTrell0()
     return  render_template('index.html', list=list)
 
-@app.route('/hello/<name>')
-def hello(name):
-   return 'welcome %s' % name
+def getAllToDoFromTrell0():
+   id = '60af9248e87283184d346aa8'
```

Comment:
This isn't a secret variable, but it's configuration that we'll want to be able to vary between different environments, so we should grab this from an environment variable like we do with your key & token

Reply:
Done